.github/workflows/tpu-testing.yml

```
@@ -0,0 +1,105 @@
+name: Run TPU tests on current PR.
+
+on:
+  pull_request:
+    branches:
+    - master
```

```suggestion
  push:
    branches:
      - master
  pull_request:
    branches:
      - master
```
it shall run also on master as the project stats are collected from amster